js/script.js

```diff
@@ -0,0 +1,12 @@
+var button = document.getElementById('addElem');
+var	list = document.getElementById('list');
+
+
+
+button.addEventListener('click', function() {
+	var newElem = document.createElement('li');
+	var itemNumber = document.getElementsByTagName('li');
```

Comment:
name of this variable suggest us that you have in this variable a number :) but you contain in this variable elements